zenodo/modules/utils/files.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
```

Comment:
Some thing went wrong with rebasing here (this file shouldn't be part of this PR)